internal/app/commands/sandbox/start.go

```diff
@@ -83,9 +67,14 @@ func StartSandbox(c *cli.Context, sandbox *Sandbox, detach, devMode, debug bool)
 		} else {
 			AskToStopSandbox(rData, force)
 		}
-	} else if !util.IsPortAvailable(8080) {
-		fmt.Fprintln(os.Stderr, "Port 8080 is not available, stop the app using it first!")
-		os.Exit(1)
+	} else {
+		ports := []uint16{httpPort, common.MGMT_PORT, common.INFO_PORT}
+		for _, port := range ports {
+			if !util.IsPortAvailable(port) {
+				fmt.Fprintf(os.Stderr, "Port %d is not available, stop the app using it first!\n", port)
+				os.Exit(1)
+			}
+		}
```

Comment:
Nice! Just a suggestion / potential improvement:  If I understand the code correctly, then this will exit after reporting the first unavailable port, regardless of how many ports are unavailable. Would it make sense to check all ports before interesting? So that in the case that the user has two unavailable ports, they'll be notified about both before the program exits? Otherwise they might end up trying to start the sandbox multiple times.

E.g. with the current implementation, the user:

1. has other apps running on ports 8080 *and* 2609.
2. tries to start the sandbox and is told port 8080 is unavailable.
3. shuts down the app that uses port 8080.
4. tries to start the sandbox again, but is told port 2609 is unavailable.
5. shuts down the app that uses port 2609
6. successfully starts the sandbox. 

Instead, if the user was alerted of multiple errors at the same time, we could skip a number of steps. That is, the user:

1. has apps running on 2609 and 8080
2. tries to start the sandbox, is told that ports 8080 and 2609 are unavailable
3. stops the apps running  on those ports
4. successfully starts the sandbox

Does that sound reasonable?

Reply:
That is actually a good point ! 👍 
Let's do it your way to avoid confusion when more than one port is unavailable !

Reply:
Awesome! 💯